### ISSUE TITLE
Empty table throws propTypes warning

### ISSUE THREAD

**Issue by dulakm** [schrodinger/fixed-data-table-2#433]
## Expected Behavior
When I render table with zero rows (`rowsCount` equal to `0`), table should be rendered without a warning.

## Current Behavior
When I render following table:
```
 <Table
    rowsCount={ 0 }
    rowHeight={ 50 }
    width={ 1000 }
    height={ 500 }
    headerHeight={ 100 }
  >
    <Column
      header={ <Cell>Header</Cell> }
      cell={ <Cell>Basic content</Cell> }
      width={ 200 }
    />
  </Table>
```
it throws following warning:
```
Warning: Failed prop type: The prop `endViewportRowIndex` is marked as required in `FixedDataTableBufferedRows`, but its value is `undefined`.
    in FixedDataTableBufferedRows (created by FixedDataTable)
    in FixedDataTable (created by FixedDataTableContainer)
    in FixedDataTableContainer
```

## Your Environment
* Version used: **1.0.0-beta.17**
* Browser Name and version: **Google Chrome (73.0.3683.103)**
* Operating System and version (desktop or mobile): **Windows 10 - Desktop**

**Comment by wcjordan** [comment]
@pradeepnschrodinger could you investigate this?

**Comment by pradeepnschrodinger** [comment]
Looks like this came up through #421, probably because I forgot to assign `endViewPortRowIndex` on the edge case where rows don't exist.

I'll open up a PR soon and update when the fix is released. Thanks for reporting.

**Comment by pradeepnschrodinger** [comment]
@dulakm, fix released with [v1.0.0-beta.18](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.0.0-beta.18) through #437.

**Comment by dulakm** [comment]
Thank you @wcjordan and @pradeepnschrodinger for such a quick reaction and fix!
